adal/src/Microsoft.IdentityModel.Clients.ActiveDirectory/Internal/ClientCreds/JsonWebToken.cs

```diff
@@ -209,18 +209,19 @@ internal class JWTPayload
         [DataContract]
         internal sealed class JWTHeaderWithCertificate : JWTHeader
         {
-            private string _thumbPrint;
-
-            public JWTHeaderWithCertificate(IClientAssertionCertificate credential)
+            public JWTHeaderWithCertificate(IClientAssertionCertificate credential, bool sendX5C)
```

Comment:
sendX5C name does not reflect usage of parameter

Reply:
@trwalke

---
In reply to: [202846673](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1124#discussion_r202846673) [](ancestors = 202846673)

Reply:
Updated parameter to "sendCertificate"

Reply:
the parameters used here are still called sendX5C adaiks

---
In reply to: [207037978](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1124#discussion_r207037978) [](ancestors = 207037978)